filetypes.orig

```diff
@@ -0,0 +1,29 @@
+# This file contains definitions of file types which will be created
```

Comment:
Whats the purpose of this file?

Reply:
Hm, actually that's superfluous and a residual of my own development branch. It's just the original contents for FILETYPES, but it's an exact copy of `filetypes.conf` and therefore can be removed. Sorry for the oversight, and thanks for spotting it!